packages/@lightningjs/ui-components/src/components/TextBox/TextBox.test.js

```diff
@@ -168,14 +168,34 @@ describe('TextBox', () => {
       expect(textBox._Text.text.textColor).toBe(getValidColor('#000000'));
     });
 
-    // TODO: Fix reference to Base
-    // it('should get TextDefaults when switching themes', async () => {
-    //   context.setTheme(Base);
-    //   textBox.base = context.theme;
-    //   textBox.content = 'Hello Text Defaults';
-    //   await textBox.__updateSpyPromise;
-    //   expect(textBox._Text.text.maxLinesSuffix).toBe('..');
-    // });
+    it('should get TextDefaults when switching themes', async () => {
+      const testTheme = {
+        name: 'Test',
+        typography: {
+          display1: {
+            fontFace: 'TestFont1',
+            fontSize: 35,
+            fontWeight: 500,
+            letterSpacing: 1,
+            lineHeight: 100,
+            verticalAlign: 'middle'
+          },
```

Comment:
Do we need this if we aren't checking for any of these values?

Reply:
expect(textBox.style.textStyle).toMatchObject(testTheme.typography.body1);
In the above test it make sure to match the object that includes everything defined in testTheme.typography.body1.
Or do you think we don't need to test this?

Reply:
No, I think that's fine! I'm referring to the `display1` object. There's no test for that, right?

Reply:
Oh, yeah. I misread it as body1. Yeah I can remove that.